.eslintrc.json

```diff
@@ -48,7 +48,13 @@
 				"@typescript-eslint/no-empty-interface": 0,
 				"@typescript-eslint/no-empty-function": 0,
 				"@typescript-eslint/no-unused-vars": "error",
-				"no-constant-condition": 0
+				"no-constant-condition": 0,
+				"max-len": [
+					"error",
+					{
+						"code": 120
+					}
+				]
```

Comment:
@adamziel before I wanted to check if you have any issues with this rule, before I fix all the linter errors.

Reply:
Will this affect doc comments?

Reply:
It will show an error when doc comments are too long. This is my main reason for doing it.

I'm unsure why it doesn't auto-wrap when linting, but I can explore how to fix it.

Reply:
[Eslint doesn't support auto fixing max-len errors.](https://github.com/eslint/eslint/issues/11325)

We would need to fix all errors manually.

Reply:
Lovely! My main concern here is only limiting the length of the comments, not actual lines of code. I've seen ample examples of the max line length rule in the Gutenberg and WordPress codebase where it actually reduced the readability. An 80 or a 120 characters long comment line seems fine, whether it's slightly indented or not. Also, ideally the actual code would be unaffected by this, so functions, object declarations etc.

Reply:
I'm not sure I understand. 

Would you like to limit only comments or comments and code?
This rule as-is limits comments and code.

The [max-len rule](https://eslint.org/docs/latest/rules/max-len) is flexible, so we could add different rules for code and comments.

Reply:
Only the length of the comments, counting from the `//`, not from the beginning of the line.

Reply:
[eslint-plugin-comment-length](https://www.npmjs.com/package/eslint-plugin-comment-length?activeTab=readme) could work for us. I will try it.

Reply:
Thank you!